.drone.yml

```diff
@@ -615,6 +616,36 @@ steps:
         exclude:
         - pull_request
 
+
+---
+kind: pipeline
+name: docker-manifest
```

Comment:
We also have an arm64 runner that you can target

Reply:
Thanks for the info, I added the build and manifest part for arm64.